src/cloud-functions.ts

```diff
@@ -322,9 +333,9 @@ export interface MakeCloudFunctionArgs<EventData> {
   dataConstructor?: (raw: Event) => EventData;
   eventType: string;
   handler?: (data: EventData, context: EventContext) => PromiseLike<any> | any;
-  labels?: { [key: string]: any };
+  labels?: Record<string, string>;
```

Comment:
Is there a risk that some projects might have non-string values here? If so, are we releasing this as a breaking change?

Reply:
The backend never accepted anything but a string.

Reply:
So my perception is - yes this is breaking but uses who used anything other than string would not have succeeded in deploying a function with non-string. This seems okay to do.